pkgs/development/python-modules/vg/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, poetry-core
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "vg";
+  version = "2.0.0";
+  format = "pyproject";
```

```suggestion
  format = "pyproject";
  disabled = pythonOlder "3.7";
```
It requires a newer Python than 3.6, so it would be good if we disable everything below 3.7 here.